### ISSUE TITLE
Make "No space left on device while finalizing archive" more useful

### ISSUE THREAD

**Issue by benapetr** [sosreport/sos#1931]
On WHICH device? Can you please add this to that error message? For example:

"No space left in /var/tmp while finalizing archive" would be thousand times more useful error message.

**Comment by pmoravec** [comment]
See preamble in terminal:

    An archive containing the collected information will be generated in
    /var/tmp/sos.JqTRsk and may be provided to a Red Hat support
    representative.

Is this sufficient?

Also, does not the full message contain the path already? See https://github.com/sosreport/sos/blob/master/sos/sosreport.py#L1234-L1235 .

**Comment by bmr-cymru** [comment]
If someone is using verbose logging then those messages will be long gone by the time the finalisation failure occurs. While everyone who uses sos regularly will "just know" where to look, I think this isn't an unreasonable ask for an abnormal end to an `sosreport` run.

That said, you actually changed this already last year @pmoravec  :-D - as you mention, the current version of the message does include the archive path:

```
commit 62aadebc49db6874aa9332a0bcf81a5fff1c066d
Author: Pavel Moravec <pmoravec@redhat.com>
Date:   Tue Jun 5 08:35:39 2018 +0200

    [sosreport] more logging when creating archive failed
    
    (Also) in case of not fatal FS error, print some valuable message
    
    Signed-off-by: Pavel Moravec <pmoravec@redhat.com>

```

This is included in sos-3.6 onwards:

```
$ git describe --contains 62aadebc4
3.6~100
```

So please update to a version that includes this change (or request that your distribution either updates the package or provides a backport of this change).

**Comment by benapetr** [comment]
well it's RedHat 7.x (I think 7.3 not sure), I don't know if newer version of RedHat 7 has that

**Comment by benapetr** [comment]
but if it's already implemented then it's fine, but I ran over this issue many times and it was always something not trivial to discover. On small and simple servers this is not a problem, but with very large and complex system (extremely large Oracle machines), where you use sophisticated FS layout, with dedicated FS for /var/log, /tmp, /var etc. these dumps on such system with huge amount of CPUs, LUNs and other peripherals are extremely large, sometimes taking many GB...

**Comment by TurboTurtle** [comment]
sos-3.6 shipped with RHEL 7.6, so the GA release of 7.6 would have this fix.
